.gitattributes

```diff
@@ -7,6 +7,7 @@
 *.java text
 *.xml text
 *.sh text
+*.properties text
```

Comment:
As a suggestion, I prefer to have changes unrelated to the issue being resolved done in a separate PR. I won't block the PR on it, it is only a recommendation.